client/src/components/Form/Form.spec.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react'
+import Form from './Form'
+
+test('loads item', () => {
```

Comment:
Please add another test, which tests that the callback function is called with the correct data.

Reply:
Added to todo-list